README.md

```diff
@@ -36,14 +38,69 @@ var Button = component({
 })
 
 // register as custom element
-Button = toCustomElement(customButton, 'button')
-document.registerElement('custom-button', Button)
+// The second parameter corresponds to a string Array containing the names of the attributes you'd like to observe and react to changes.
+var CustomButton = toCustomElement(Button, ['title'])
+document.registerElement('custom-button', CustomButton)
 
 // create new custom-button
-var button = document.createElement('button', 'custom-button')
+var button = document.createElement('custom-button')
 document.body.appendChild(button)
 ```
 
+## Custom Elements (webcomponents-v1)
+```js
+var toCustomElementV1 = require('nanocomponent-adapters/custom-element-v1')
+var component = require('nanocomponent')
+var html = require('bel')
+
+// create new nanocomponent
+var Button = component({
+  render: function (data) {
+    return html`
+      <button>hello planet</button>
+    `
+  }
+})
+
+// register as custom element
+// The second parameter corresponds to a string Array containing the names of the attributes you'd like to observe and react to changes.
+var CustomButton = toCustomElement(Button, ['title']);
+customElements.define('custom-button', CustomButton);
+// create new custom-button
+var button = document.createElement('custom-button')
+document.body.appendChild(button)
+```
+
+## Angular
+```js
```

Comment:
Yeah, the adapter provides that generic interface. The Angular (2+) example uses the Custom Element produced and integrates it into its template. In that way, the angular example shouldn't be generic. Should be specific on how to let know Angular about your custom element. In this case you only need to use the CUSTOM_ELEMENTS schema and provide it to your module.

And yeah, there are ways but sticking to the standard is always better. You can still use v0 if you don't want to use the class syntax. Also, every browser that implements custom elements also implements the class syntax.